api/support/verify_state.py

```
@@ -0,0 +1,289 @@
+            metrics_text: str = loop.run_until_complete(_verify_all())
+            if metrics_text:
+                loop.run_until_complete(_push_to_grafana(metrics_text))
+
+            self.send_response(200)
+            self.send_header("Content-type", "text/plain")
+            self.end_headers()
+            response_body = (
+                f"verify_state completed\n\nMetrics:\n{metrics_text}".encode()
+            )
+            self.wfile.write(response_body)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)